index.js

```diff
@@ -4,22 +4,26 @@ var Botkit = require('botkit')
 var Store = require('jfs')
 var beep = require('beepboop-botkit')
 var NoLimit = require('nolimit')
-var nolimit = new NoLimit({ filename: 'sirdibsabot'})
-var token = process.env.SLACK_TOKEN
-var postChannel = process.env.MAIN_CHANNEL
-var admin1 = process.env.ADMIN1
-var admin2 = process.env.ADMIN2
-var filename = process.env.FILENAME
+var nolimit = new NoLimit({ filename: process.env.FILE_NAME })
+var params = {
+  token: process.env.SLACK_TOKEN,
+  postChannel: process.env.MAIN_CHANNEL,
+  admin: {
+    primary: process.env.ADMIN_PRIMARY_USER,
+    secondary: process.env.ADMIN_SECONDARY_USER
+  },
+  fileName: process.env.FILE_NAME
+}
 var controller = Botkit.slackbot({
   retry: Infinity,
   debug: false
 })
 
 // Assume single team mode if we have a SLACK_TOKEN
-if (token) {
+if (params.token) {
   console.log('Starting in single-team mode')
   controller.spawn({
-    token: token
+    token: params.token
```

Comment:
this can't be added to the env file, must stay separate its value is via beep boop